### ISSUE TITLE
Text that is hyperlinked in a "note" callout fails accessibility checks

### ISSUE THREAD

**Issue by phrdang** [berkeley-cdss/berkeley-class-site#65]
Text that is hyperlinked in a Just the Docs "note" callout fails accessibility checks due to having not enough color contrast. This may be an issue with the way axe lints this because the background and foreground colors seem incorrect? Not sure.

Example Markdown text that causes the error:

```md
{: .note }
If sections already exist, reimporting them will result in [duplicate sections](https://github.com/Cal-CS-61A-Staff/berkeley-cs61a/issues/7788). See [Updating existing sections](#updating-existing-sections).
```

Example failed PR job run: https://github.com/berkeley-cdss/docs/actions/runs/17559665899/job/49872617670?pr=33

**Comment by pancakereport** [comment]
I just checked your example and I'm not seeing an a11y violation. I think Brandon probably fixed this in https://github.com/berkeley-cdss/berkeley-class-site/pull/70 please reopen or file another issue if you find that this is still a problem.
